_CodingChallenges/94-2048

```diff
@@ -0,0 +1,14 @@
+---
+title: "2048"
+video_number: 94
+date: 2018-02-16
+video_id: 
```

Comment:
This will cause problems when Jekyll tries to build the site. Please put the video's ID here (when it gets released).